superdesk/newsroom-core
# [CPCN-178] Implement a new report for Company Admins

### Purpose
Implement a new report for Company Admins. The report shows the number of saved My topics and Company topics per user in their company.

### What has changed
- Added the new report
- Completed some typings in the frontend code
- Small adjustment to `__REDUX_DEVTOOLS_EXTENSION_COMPOSE__` usage so store name will be recognised in the browser terminal
- Adjusted template filters according to user type 

### How to test
- Checkout the branch and run the project
- Log in as a Company Admin user and observe that the user is now able to see the reports section 
- Enter the reports section and select the only report in the dropdown an run it
- Please double check if the calculations are correct

### Checklist

### Screenshots
This is how it looks
<img width="750" alt="image" src="https://github.com/superdesk/newsroom-core/assets/124766/7a7c9b08-d0db-4405-9206-c86887d1e218">

~**Note:** I will add a few tests cases tomorrow.~

Thanks for checking!

Resolves: [CPCN-178]

[CPCN-178]: https://sofab.atlassian.net/browse/CPCN-178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
